pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -401,10 +400,10 @@ func compatibleRoutesForHost(routes []*istio.TLSRoute, parentHost string) []*ist
 	return res
 }
 
-func routeMeta(obj controllers.Object) map[string]string {
-	m := parentMeta(obj, nil)
-	m[constants.InternalRouteSemantics] = constants.RouteSemanticsGateway
-	return m
+func routeMeta(controllers.Object) map[string]string {
+	return map[string]string{
```

Comment:
To make sure I'm understanding this correctly: we don't inherit the parent meta anymore because the events are now sent on the converted objects (e.g. VS)?

Reply:
I'm going to revert this (as I commented but forgot to push) but this is just removing the parent annotation. That was added so we could do a lookup on the VS to the HTTPRoute to match an event like "HTTPRoute X changed". Now we have a VS and get a "VS changed" event so we don't need it. 

But will still keep it since it's handy for debugging